apps/client-server/src/app/post/post-manager.service.spec.ts

```diff
@@ -107,7 +113,7 @@ describe('PostManagerService', () => {
     return dto;
   }
 
-  afterAll(async () => {
+  afterEach(async () => {
     await orm.close(true);
     await module.close();
   });
```

Comment:
Still doesn't work, sadly. It says that orm is undefined, the same thing to do with modules and dependencies as above

Reply:
Please revert here, I will worry about fixing the tests.
Unfortunately it is a little scuffed to test here so usually I just wrap the in code in a try/catch and run the debugger to see what error is actually being thrown. Mocks would be more efficient for some of these features that don't need as much real data.